tools/wheels/cibw_before_build.sh

```diff
@@ -16,54 +19,36 @@ elif [[ $RUNNER_OS == "Windows" ]]; then
     cat $PROJECT_DIR/tools/wheels/LICENSE_win32.txt >> $PROJECT_DIR/LICENSE.txt
 fi
 
-# the macos_arm64 build might set this variable
-if [ -z $INSTALL_OPENBLAS ]; then
+if [[ $(python -c"import sys; print(sys.maxsize)") < $(python -c"import sys; print(2**33)") ]]; then
+    echo "No BLAS used for 32-bit wheels"
+    export INSTALL_OPENBLAS=false
+elif [ -z $INSTALL_OPENBLAS ]; then
+    # the macos_arm64 build might not set this variable
     export INSTALL_OPENBLAS=true
 fi
 
-# Install Openblas
-if [[ $RUNNER_OS == "Linux" || ($RUNNER_OS == "macOS" && ("$INSTALL_OPENBLAS" = true)) ]] ; then
-      basedir=$(python tools/openblas_support.py --use-ilp64)
-    if [[ $RUNNER_OS == "macOS" && $PLATFORM == "macosx-arm64" ]]; then
-        # /usr/local/lib doesn't exist on cirrus-ci runners
-        sudo mkdir -p /usr/local/lib /usr/local/include /usr/local/lib/cmake/openblas
-        sudo mkdir -p /opt/arm64-builds/lib /opt/arm64-builds/include
-        sudo chown -R $USER /opt/arm64-builds
-        cp -r $basedir/lib/* /opt/arm64-builds/lib
-        cp $basedir/include/* /opt/arm64-builds/include
-        sudo cp -r $basedir/lib/* /usr/local/lib
-        sudo cp $basedir/include/* /usr/local/include
-    else
-        cp -r $basedir/lib/* /usr/local/lib
-        cp $basedir/include/* /usr/local/include
-    fi
-elif [[ $RUNNER_OS == "Windows" ]]; then
+# Install Openblas from scipy-openblas64
+if [[ "$INSTALL_OPENBLAS" = "true" ]] ; then
+    echo PKG_CONFIG_PATH $PKG_CONFIG_PATH
+    PKG_CONFIG_PATH=$PROJECT_DIR/.openblas
+    rm -rf $PKG_CONFIG_PATH
+    mkdir -p $PKG_CONFIG_PATH
+    python -m pip install scipy-openblas64
```

Comment:
for maintenance branches how do we choose what version of scipy-openblas gets installed. e.g. for a certain maintenance branch we may want to stick with a certain (older) version, and bump when we do a major release. At the moment we just get the latest version.

Reply:
This should use the ci_requirements.txt file.